src/kernel/capinv.c

```diff
@@ -1531,6 +1550,19 @@ composite_syscall_slowpath(struct pt_regs *regs, int *thd_switch)
 
 			break;
 		}
+		case CAPTBL_OP_TCAP_WAKEUP:
+		{
+			struct cap_tcap *tcapwkup = (struct cap_tcap *)ch;
```

Comment:
I don't really care for this PR itself, but what does the style guide say for underscores here? I would have expected tcap_wkup.